seawatch_planner/settings/dev.py

```diff
@@ -25,7 +29,7 @@
     dj_database_url.config(default='postgres://' + db_user + ':@' + db_host + ':' + str(db_port) + '/' + db_name,
                            ssl_require=False)
 
-
 NOSE_ARGS = ['--nocapture',
              '--nologcapture']
 
+DEBUG_PROPAGATE_EXCEPTIONS = True
```

Comment:
What does this do, why do we need it?

Reply:
We don't need it it was there for debugging purposes. So i will delete it.